### DISCUSSION TITLE
Contact disappears after deleting the photo

### DISCUSSION THREAD

**Discussion by WebworkrNet** [FossifyOrg/General-Discussion#662] [General]
I had edited some fields of an existing contact, but had not saved them temporarily. Then I removed the photo. Suddenly the contact disappeared. I seem to remember that the app also crashed in this context. Has this ever happened to anyone else? It can't be found in the Android recycle bin either. Does anyone else have any idea where the contact could be found? I have already tried various search terms. Well, maybe it can still be found.

I have not deleted the phone book entry regularly. There is still a confirmation prompt here. I would certainly have noticed that.

The damage is limited in this case. It was a duplicate contact that I wanted to overwrite.

Thanks

**Comment by WebworkrNet** [comment]
It doesn't seem to have been an isolated case, the error has just been reproduced. I'll keep an eye on it. However, the device currently has extremely little free memory. I will check the situation again once this has been rectified. If the phenomenon still occurs, I will open a regular bug report.

**Comment by naveensingh** [comment]
> If the phenomenon still occurs, I will open a regular bug report.

If it is still happening, please do.
